<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendEditDialog</class>
 <widget class="QDialog" name="FriendEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="e_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>PhoneNumber</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="e_phoneNumber"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="e_email"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>BirthYear</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="e_birthYear"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="e_title">
     <item>
      <property name="text">
       <string>Mr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mrs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Miss</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>HomeNumber</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="e_homeNumber"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FriendEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FriendEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
